src/map/battle.c

```diff
@@ -4695,6 +4693,7 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 					break;
 				status_change_end(src, SC_AUTOCOUNTER, INVALID_TIMER);
 				FALLTHROUGH
+#ifdef RENEWAL
```

Comment:
Any source for this change? why are these skill changes being disabled in pre-re.

Reply:
Sorry, wrong section.